src/FTPClient.jl

```diff
@@ -16,6 +16,7 @@ type FTPClientError <: Exception
        lib_curl_error::UInt32
 end
 
+@enum FTP_MODES ascii_mode binary_mode
```

Comment:
Based upon the `@enum` documentation I think this should be named `FTP_MODE` rather than a plural